src/include/sof/audio/drc/drc_math.h

```diff
@@ -59,9 +60,7 @@ static inline int32_t drc_sin_fixed(int32_t x)
 {
 	const int32_t lshift = drc_get_lshift(30, 30, 28);
 	int32_t denorm_x = drc_mult_lshift(x, PI_OVER_TWO_Q30, lshift);
-	int32_t sin_val = sin_fixed_16b(denorm_x);
-
-	return sin_val << 16;
+	return sofm_lut_sin_fixed_32b(denorm_x);
```

Comment:
some of the tooling might complain about a missing line between variable definitions and statements - same below